Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -182,20 +182,14 @@ def get_feed_content_divided_to_lines(self, url, raw_response):
 
 
 def determine_indicator_type(indicator_type, default_indicator_type, value):
-    if not indicator_type:
+    if not indicator_type and default_indicator_type:
         indicator_type = default_indicator_type
-    if indicator_type == FeedIndicatorType.Domain and '*' in value:
-        indicator_type = FeedIndicatorType.DomainGlob
+    elif not indicator_type and not default_indicator_type:
+        indicator_type = auto_detect_indicator_type(value)
     return indicator_type
```

Comment:
You're not actually using the parameter, `auto_detect_type`, that you've added to the YAML file anywhere. If the option is true, we should ignore whatever indicator_type was passed and just do auto-detection. Need to update your conditions to reflect that.